jwt-client/jwt-client-spring-core/src/main/java/org/entur/jwt/client/properties/JwtClientProperties.java

```diff
@@ -9,6 +9,8 @@ public class JwtClientProperties {
 
     protected Map<String, KeycloakJwtClientProperties> keycloak = new HashMap<>();
 
+    protected Map<String, GenericJwtClientProperties> generic = new HashMap<>();
```

Comment:
Up for discussion whether `generic` is the best naming in the properties which looks like this:
```
entur.jwt.clients.generic.myClient.client-id=myClientId
```
Alternative could be `oauth2` 🤔

Reply:
Hmm I think as we already have Auth0 and Keycloak, that `generic` is the best alternative, i.e. Auth0 and Keycloak could also be oauth2.